views/js/pciCreator/ims/likertCompact/imsPciCreator.json

```diff
@@ -4,7 +4,7 @@
     "label": "IMS Likert Scale Compact v1",
     "short": "Likert(C) v1",
     "description": "A simple implementation of likert scale.",
-    "version": "1.2.0",
+    "version": "1.3.0",
```

Comment:
I don't see any change on the IMS version the PCI. Should you update it as well or only the TAO version?